File: 3830-find-closest-person/find-closest-person.cs
public class Solution {
    public int FindClosest(int x, int y, int z) {
        int first=0,second=0;
        if(x>z){
            first=x-z;
        }
        else if(x<z){
            first=z-x;
        }
        if(y>z){
            second=y-z;
        }
        else if(y<z){
            second=z-y;
        }
        if(first>second){
            return 2;
        }
        else if(first<second){
            return 1;
        }
        else{
            return 0;
        }
    }
}

File: 3830-find-closest-person/find-closest-person.py
class Solution(object):
    def findClosest(self, x, y, z):
        """
        :type x: int
        :type y: int
        :type z: int
        :rtype: int
        """
        first,second=0,0
        if x>=z:
            first=x-z
        elif x<z:
            first=z-x
        if y>=z:
            second=y-z
        elif y<z:
            second=z-y
        if first<second: return 1
        elif first>second: return 2
        else: return 0